CHANGELOG.md

```diff
@@ -38,7 +38,7 @@
 - [#5479](https://github.com/influxdata/influxdb/issues/5479): Bringing up a node as a meta only node causes panic
 - [#5504](https://github.com/influxdata/influxdb/issues/5475): create retention policy on unexistant DB crash InfluxDB
 - [#5505](https://github.com/influxdata/influxdb/issues/5505): Clear authCache in meta.Client when password changes.
-
```

Comment:
This needs to go under the `v0.11.0` section now.  `v.0.10.0` has already been released.